varbase.install

```diff
@@ -277,4 +278,15 @@ function varbase_update_8003() {
     \Drupal::service('module_installer')->install(['link_attributes'], FALSE);
   }
 
+  // Add Field Description
+  $profile_path = drupal_get_path('profile', 'varbase') . '/config/install/';
+  $configs['field.storage.block_content.field_description'] = $profile_path . 'field.storage.block_content.field_description.yml';
+  $configs['field.storage.node.field_description'] = $profile_path . 'field.storage.node.field_description.yml';
+  $configs['field.storage.taxonomy_term.field_description'] = $profile_path . 'field.storage.taxonomy_term.field_description.yml';
+
+  foreach ($configs as $name => $config_path) {
+    $data = Yaml::parse($config_path);
```

Comment:
Have data been converted to an array?
I had some issues with this not been converted to array.